src/containerapp/azext_containerapp/_params.py

```diff
@@ -0,0 +1,186 @@
+        c.argument('container_name', type=str, help="Name of the container.")
+        c.argument('cpu', type=float, validator=validate_cpu, help="Required CPU in cores, e.g. 0.5")
+        c.argument('memory', type=str, validator=validate_memory, help="Required memory, e.g. 1.0Gi")
+        c.argument('env_vars', nargs='*', help="A list of environment variable(s) for the container. Space-separated values in 'key=value' format. Empty string to clear existing values. Prefix value with 'secretref:' to reference a secret.")
+        c.argument('startup_command', nargs='*', options_list=['--command'], help="A list of supported commands on the container that will executed during startup. Space-separated values e.g. \"/bin/queue\" \"mycommand\". Empty string to clear existing values")
+        c.argument('args', nargs='*', help="A list of container startup command argument(s). Space-separated values e.g. \"-c\" \"mycommand\". Empty string to clear existing values")
+        c.argument('revision_suffix', type=str, help='User friendly suffix that is appended to the revision name')
+
+    # Env vars
+    with self.argument_context('containerapp', arg_group='Environment variables (Creates new revision)') as c:
+        c.argument('set_env_vars', nargs='*', help="A list of environment variable(s) to add to the container. Space-separated values in 'key=value' format. If stored as a secret, value must start with \'secretref:\' followed by the secret name.")
+        c.argument('remove_env_vars', nargs='*', help="A list of environment variable(s) to remove from container. Space-separated env var name values.")
+        c.argument('replace_env_vars', nargs='*', help="A list of environment variable(s) to replace from the container. Space-separated values in 'key=value' format. If stored as a secret, value must start with \'secretref:\' followed by the secret name.")
+        c.argument('remove_all_env_vars', help="Option to remove all environment variable(s) from the container.")
+
+    # Scale
+    with self.argument_context('containerapp', arg_group='Scale (Creates new revision)') as c:
+        c.argument('min_replicas', type=int, help="The minimum number of replicas.")
+        c.argument('max_replicas', type=int, help="The maximum number of replicas.")
+
+    # Dapr
+    with self.argument_context('containerapp', arg_group='Dapr') as c:
+        c.argument('dapr_enabled', options_list=['--enable-dapr'], default=False, arg_type=get_three_state_flag(), help="Boolean indicating if the Dapr side car is enabled.")
+        c.argument('dapr_app_port', type=int, help="The port Dapr uses to talk to the application.")
+        c.argument('dapr_app_id', type=str, help="The Dapr application identifier.")
+        c.argument('dapr_app_protocol', type=str, arg_type=get_enum_type(['http', 'grpc']), help="The protocol Dapr uses to talk to the application.")
+        c.argument('dapr_components', help="The name of a yaml file containing a list of dapr components.")
```

Comment:
This is probably no longer needed

```suggestion
        c.argument('dapr_components', help="The name of a yaml file containing a list of Dapr components.")
```